lib.php

```diff
@@ -244,6 +244,7 @@ public static function save_config_options(Pieform $form, $values) {
     }
 
     public static function postinst($fromversion) {
```

Comment:
Not 100% sure, but is this called when you upgrade the plugin? Or only on install?

If its called on upgrade we need a check for if the config already exists so we dont overwrite anything :smile:

Reply:
It's called on both install and upgrade. To call only on install you can add an `if ($fromversion == 0)` 
However, I think an empty check is simpler to handle both install and upgrade.

Reply:
So if someone already has the plugin installed, then wouldn't this override their config on every upgrade ? Or am I missing something

Reply:
Ah NVM i see you added a check